device/globals/hotspot.gd

```diff
@@ -0,0 +1,86 @@
+extends "res://globals/interactive.gd"
+
+export(String, FILE, ".esc") var events_path
+export var global_id = ""
+export var tooltip = ""
+
+var event_table = {}
+
+func get_tooltip():
+	if TranslationServer.get_locale() == ProjectSettings.get_setting("escoria/platform/development_lang"):
+		if not global_id and ProjectSettings.get_setting("escoria/platform/force_tooltip_global_id"):
+			vm.report_errors("exit", ["Missing global_id in exit with tooltip '" + tooltip + "'"])
+		return tooltip
+
+	var tooltip_identifier = global_id + ".tooltip"
+	var translated = tr(tooltip_identifier)
+
+	if translated == tooltip_identifier:
+		if not global_id and ProjectSettings.get_setting("escoria/platform/force_tooltip_global_id"):
+			vm.report_errors("exit", ["Missing global_id in exit with tooltip '" + tooltip + "'"])
+		return tooltip_identifier
+
+	return translated
+
+func mouse_enter():
+	vm.hover_begin(self)
+	var tt = get_tooltip()
+	var text = tr(tt)
+	get_tree().call_group_flags(SceneTree.GROUP_CALL_DEFAULT, "hud", "set_tooltip", text)
+
+func mouse_exit():
+	vm.hover_end()
+	get_tree().call_group_flags(SceneTree.GROUP_CALL_DEFAULT, "hud", "set_tooltip", "")
+
+func area_input(viewport, event, shape_idx):
+	input(event)
+
+func input(event):
+	if !(event is InputEventMouseButton and event.is_pressed()):
+		return
+
+	var player = vm.get_object("player")
+	# Get mouse position, since event.position only works with Area2D exits
+	var pos = get_viewport().get_mouse_position()
+	printt("exit pos", pos)
+
+	if player and event.doubleclick:
+		player.set_position(pos)
+	elif player:
+		# Control blocks input to background, so make player walk
+		player.walk_to(pos)
```

Comment:
As mentioned on IRC, this is probably wrong if the background is larger than the viewport, ie. the player will walk to the wrong position.

Reply:
That doesn't have to be solved by this PR though, as long as there's an issue for it.

Reply:
Breaks with `event.position` too, #121 is close enough for this issue, or what do you think?

It's probably symptoms of the same problem.

Reply:
Yes, I agree.